devsetup/Makefile

```diff
@@ -83,6 +84,41 @@ crc_cleanup: ## Destroys the CRC env, but does NOT clear ( --clear-cache ) the c
 	sudo ${CLEANUP_DIR_CMD} /etc/pki/ca-trust/source/anchors/crc-router-ca.pem
 	sudo update-ca-trust
 
+.PHONY: add_nics_crc
+add_nics_crc: remove_nics_crc
+	# Add data interface
+	MAC_ADDRESS=$(shell echo -n 52:54:00; dd bs=1 count=3 if=/dev/random 2>/dev/null | hexdump -v -e '/1 "-%02X"' | tr '-' ':'); \
+	virsh --connect=qemu:///system attach-interface --domain crc --type network --source data --model virtio --mac $$MAC_ADDRESS --config;
+	sleep 1
+	# Add management interface
+	MAC_ADDRESS=$(shell echo -n 52:54:00; dd bs=1 count=3 if=/dev/random 2>/dev/null | hexdump -v -e '/1 "-%02X"' | tr '-' ':'); \
+	virsh --connect=qemu:///system attach-interface --domain crc --type network --source management --model virtio --mac $$MAC_ADDRESS --config;
+	sleep 1
+	# Add external interface
+	MAC_ADDRESS=$(shell echo -n 52:54:00; dd bs=1 count=3 if=/dev/random 2>/dev/null | hexdump -v -e '/1 "-%02X"' | tr '-' ':'); \
+	virsh --connect=qemu:///system attach-interface --domain crc --type network --source external --model virtio --mac $$MAC_ADDRESS --config;
+	sleep 1
+	virsh --connect=qemu:///system destroy crc
+	timeout --foreground ${TIMEOUT_VALUE} bash -c 'until $$(virsh --connect=qemu:///system list --all | grep crc | grep -q "shut off"); do sleep 1; done'
+	virsh --connect=qemu:///system start crc
+	timeout --foreground ${TIMEOUT_VALUE} bash -c 'until $$(virsh --connect=qemu:///system list --all | grep crc | grep -q "running"); do sleep 1; done'
+
+.PHONY: remove_nics_crc
+remove_nics_crc:
+	-MAC_ADDRESS=$(shell virsh --connect=qemu:///system dumpxml crc | grep "source network='data'" -B1 | grep mac | cut -d"'" -f2); \
+	virsh --connect=qemu:///system detach-interface --domain crc --type network --mac $$MAC_ADDRESS --config
+	sleep 1
+	-MAC_ADDRESS=$(shell virsh --connect=qemu:///system dumpxml crc | grep "source network='management'" -B1 | grep mac | cut -d"'" -f2); \
+	virsh --connect=qemu:///system detach-interface --domain crc --type network --mac $$MAC_ADDRESS --config
+	sleep 1
+	-MAC_ADDRESS=$(shell virsh --connect=qemu:///system dumpxml crc | grep "source network='external'" -B1 | grep mac | cut -d"'" -f2); \
+	virsh --connect=qemu:///system detach-interface --domain crc --type network --mac $$MAC_ADDRESS --config
+	sleep 1
+	virsh --connect=qemu:///system destroy crc
```

Comment:
you seem to repeat code here and above. should be a target `crc_restart`?

Reply:
This was on my scope, to set it as a target and only call it if some changes were made on the VM. I guess I'll handle this on a script